src/view/com/composer/Composer.tsx

```diff
@@ -769,6 +856,8 @@ function ComposerTopBar({
             <ButtonText style={[a.text_md]}>
               {isReply ? (
                 <Trans context="action">Reply</Trans>
```

Comment:
theoretically Reply should also have something ("Reply All"?) but i imagine it would be too confusing to translate. meh.